Engine/Source/Runtime/D3D12RHI/Private/D3D12PipelineState.h

```diff
@@ -46,6 +46,10 @@ struct FD3D12_GRAPHICS_PIPELINE_STATE_DESC
 	D3D12_SHADER_BYTECODE DS;
 	D3D12_SHADER_BYTECODE HS;
 	D3D12_SHADER_BYTECODE GS;
+	//AMCHANGE_begin
+    //#AMCHANGE Fix for crash on servers, this field is defined in DX12 API, but wasn't included in the ue4 implementation of it. Adding it fixed the crash
```

Comment:
Do we have any idea why this wasn't included in the unreal implementation? Did you figure this out from a forum post? A bit more context would be interesting to have in the comments.

Reply:
Added some additional comments. please review again